x-pack/test/apm_api_integration/tests/transactions/transactions_groups_detailed_statistics.spec.ts

```diff
@@ -172,51 +174,96 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           const transactionsCurrentPeriod =
             transactionsStatistics.currentPeriod[transactionNames[0]];
-          const metricsCurrentPeriod =
-            metricsStatisticsOneMinute.currentPeriod[transactionNames[0]];
+          const metricsOneMinPeriod = metricsStatisticsOneMinute.currentPeriod[transactionNames[0]];
+          const metricsTenMinPeriod = metricsStatisticsTenMinute.currentPeriod[transactionNames[0]];
+          const metricsSixtyMinPeriod =
+            metricsStatisticsSixtyMinute.currentPeriod[transactionNames[0]];
           const transactionsLatencyMean = meanBy(transactionsCurrentPeriod.latency, 'y');
-          const metricsLatencyMean = meanBy(metricsCurrentPeriod.latency, 'y');
-          [transactionsLatencyMean, metricsLatencyMean].forEach((value) =>
-            expect(value).to.be.equal(1000000)
-          );
+          const metricsOneMinLatencyMean = meanBy(metricsOneMinPeriod.latency, 'y');
+          const metricsTenMinLatencyMean = meanBy(metricsTenMinPeriod.latency, 'y');
+          const metricsSixtyMinLatencyMean = meanBy(metricsSixtyMinPeriod.latency, 'y');
+          [
+            transactionsLatencyMean,
+            metricsOneMinLatencyMean,
+            metricsTenMinLatencyMean,
+            metricsSixtyMinLatencyMean,
+          ].forEach((value) => expect(value).to.be.equal(1000000));
         });
 
         it('has same error rate mean value for metrics and transactions data', () => {
           const transactionsCurrentPeriod =
             transactionsStatistics.currentPeriod[transactionNames[0]];
-          const metricsCurrentPeriod =
-            metricsStatisticsOneMinute.currentPeriod[transactionNames[0]];
+          const metricsOneMinPeriod = metricsStatisticsOneMinute.currentPeriod[transactionNames[0]];
+          const metricsTenMinPeriod = metricsStatisticsTenMinute.currentPeriod[transactionNames[0]];
+          const metricsSixtyMinPeriod =
+            metricsStatisticsSixtyMinute.currentPeriod[transactionNames[0]];
 
           const transactionsErrorRateMean = meanBy(transactionsCurrentPeriod.errorRate, 'y');
-          const metricsErrorRateMean = meanBy(metricsCurrentPeriod.errorRate, 'y');
-          [transactionsErrorRateMean, metricsErrorRateMean].forEach((value) =>
-            expect(asPercent(value, 1)).to.be.equal(`${GO_PROD_ERROR_RATE}%`)
-          );
+
+          const metricsOneMinErrorRateMean = meanBy(metricsOneMinPeriod.errorRate, 'y');
+          const metricsTenMinErrorRateMean = meanBy(metricsTenMinPeriod.errorRate, 'y');
+          const metricsSixtyMinErrorRateMean = meanBy(metricsSixtyMinPeriod.errorRate, 'y');
+
+          [
+            transactionsErrorRateMean,
+            metricsOneMinErrorRateMean,
+            metricsTenMinErrorRateMean,
+            metricsSixtyMinErrorRateMean,
+          ].forEach((value) => expect(asPercent(value, 1)).to.be.equal(`${GO_PROD_ERROR_RATE}%`));
```

Comment:
Can this be simplified?

```suggestion
          ].forEach((value) => expect(value).to.be.equal(GO_PROD_ERROR_RATE));
```